plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -562,10 +562,28 @@ public void testTimestampWithTimeZone()
                 // max value in BigQuery
                 .addRoundTrip("TIMESTAMP", "TIMESTAMP '9999-12-31 23:59:59.999999-00:00'",
                         TIMESTAMP_TZ_MICROS, "TIMESTAMP '9999-12-31 23:59:59.999999 UTC'")
+                .execute(getQueryRunner(), trinoCreateAsSelect("test.timestamp_tz"))
```

Comment:
Please test with several timezones. You can refer to `TestPostgreSqlTypeMapping#sessionZonesDataProvider`.

Reply:
I guess here you are thinking in following the same pattern